doc/api/http2.md

```diff
@@ -3150,13 +3150,16 @@ will result in a [`TypeError`][] being thrown.
 added: v8.4.0
 -->
 * `headers` {HTTP/2 Headers Object} An object describing the headers
-* `callback` {Function}
-
-Call [`http2stream.pushStream()`][] with the given headers, and wraps the
-given newly created [`Http2Stream`] on `Http2ServerResponse`.
+* `callback` {Function} Called once `http2stream.pushStream()` is finished,
+or either when the attempt to create the pushed Http2Stream has failed or has
```

Comment:
1. `Http2Stream` -> `` `Http2Stream` ``
2. We usually align parameter description lines, so this and the next line need 2 space indent.

Reply:
For 1st:
Here the `http2Stream` is the reference to the `http2Stream.pushStream()`. And on the other hand, this lower case of `http2Stream` may be instance of `Http2Stream`. So it should be in the lower case IMO ?

For 2nd:
OK, fixed.

Reply:
For 1st: I've only meant to add backticks)

Reply:
ok, I've found it. Thanks!